packages/linear-genome-view/src/BasicTrack/components/BlockBasedTrack.tsx

```diff
@@ -66,7 +67,9 @@ function BlockBasedTrack(props: {
       className={classes.track}
       onContextMenu={event => {
         event.preventDefault()
-        if (ref.current) {
+        if (contextCoord) {
```

Comment:
what is the logic here? if there was already a context menu, make it go away?

Reply:
could you maybe add a comment to this effect here

Reply:
done